quickstarts/microsoft.cache/redis-enterprise-vectordb/main.bicep

```diff
@@ -0,0 +1,44 @@
+@description('Location of all resources')
+param location string = resourceGroup().location
+
+@description('Name of the Redis Enterprise Cache')
+param redisCacheName string = ''
```

Comment:
Can't have a default value that will fail the deployment

Reply:
I have updated the value. Please review.